app/controllers/admin/merchants_controller.rb

```diff
@@ -13,4 +16,19 @@ def update
     merchant.save
     redirect_to admin_merchants_path
   end
+
+  def downgrade
```

Comment:
This all looks good, however, for a re-factor we might make `downgrade` and `upgrade` into a more RESTful `update`